scripts/setup.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
```

Comment:
Should we add `set -e` right after the shebang? (Fail fast if any command fails)

Had this as a rule of thumb in all my bash scripts 😄